package.json

```
@@ -2,11 +2,13 @@
 	"name": "faucet-pipeline-static",
 	"version": "2.1.0",
 	"description": "static files for faucet-pipeline",
-	"main": "index.js",
+	"main": "lib/index.js",
+	"type": "module",
 	"scripts": {
-		"test": "npm-run-all --parallel lint test:cli",
+		"test": "npm run lint && npm run typecheck && npm run test:cli",
 		"test:cli": "./test/run",
-		"lint": "eslint --cache index.js test && echo ✓"
+		"lint": "eslint --cache lib test && echo ✓",
```

See above; I think this improves readability a little:
```suggestion
		"lint": "eslint --cache ./lib ./test && echo ✓",
```

---

:heavy_check_mark: